storybook/testsrunner.cpp

```diff
@@ -0,0 +1,47 @@
+#include "testsrunner.h"
+
+#include <QDir>
+#include <QProcess>
+
+TestsRunner::TestsRunner(const QString& testRunnerExecutablePath,
+                         const QString& testsBasePath, QObject* parent)
+    : QObject{parent}, m_testRunnerExecutablePath{testRunnerExecutablePath},
+      m_testsBasePath{testsBasePath}
+{
+}
+
+int TestsRunner::testsCount(const QString& fileName)
+{
+    QStringList arguments;
+    arguments << "-functions";
+    arguments << "-input" << m_testsBasePath + QDir::separator() + fileName;
+
+    QProcess testRunnerProcess;
+    testRunnerProcess.start(m_testRunnerExecutablePath, arguments);
+    testRunnerProcess.waitForFinished();
+
+    if (testRunnerProcess.exitCode())
+        return 0;
+
+    QByteArray functions = testRunnerProcess.readAllStandardError();
+    return functions.count('\n');
```

Comment:
But this sounds problematic to anything else than Linux :)

Reply:
Yeah, good point. To fix that I started the process via `open` with `QIODevice::Text` mode. According to https://doc.qt.io/qt-6/qiodevicebase.html#OpenModeFlag-enum it should cause translation of native newlines to `\n`.

Reply:
Yup, good idea!